### PR TITLE
app-antivirus/clamtk: new version of nautilus plugin to support python3

### PR DESCRIPTION
**Pull request by j-licht**
Bug: https://bugs.gentoo.org/710106
Package-Manager: Portage-2.3.84, Repoman-2.3.20
Signed-off-by: Jonas Licht <jonas.licht@fem.tu-ilmenau.de>

This depends on #14696 which adds python3 support for dev-python/nautilus-python.
I expect that the CI QA tests will fail until this wasn't merged.

### CODE REVIEW

File: app-antivirus/clamtk/clamtk-6.02-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
```
Comment (ConiKost) [inline app-antivirus/clamtk/clamtk-6.02-r2.ebuild]: Could you also test with python3_8?

**Comment by j-licht** [comment]
@ConiKost I've tested with all three python version and it basically work.
But on filenames with spaces, it opened not the scanreport but the start screen.
Probably it couldn't find the file.
Removing this quote solves the problem: https://github.com/dave-theunsub/clamtk-gnome/blob/3c0711e3c87ef804c6445e3c7e972fcb3b923bed/clamtk-gnome.py#L51

**Comment by ConiKost** [comment]
> @ConiKost I've tested with all three python version and it basically work.

Great, thanks!

> But on filenames with spaces, it opened not the scanreport but the start screen.
Probably it couldn't find the file. Removing this quote solves the problem: https://github.com/dave-theunsub/clamtk-gnome/blob/3c0711e3c87ef804c6445e3c7e972fcb3b923bed/clamtk-gnome.py#L51

But this is independend of the python version? Could you report that to upstream?

**Comment by j-licht** [comment]
> But this is independend of the python version? Could you report that to upstream?

Yes, it's independend of the python version.
Bug is already open https://github.com/dave-theunsub/clamtk-gnome/issues/15

**Comment by j-licht** [comment]
New version 0.05 of the clamtk-gnome fixes the bug.

**Comment by ConiKost** [comment]
Since nautilius is now with python3 enabled, could you rebase to newest 6.03?
Once done, I can merge :-)

**Pull request closed without merging**